inscriptions/ISic000571.xml

```diff
@@ -185,36 +190,40 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="EDR" n="EDR138359" xml:lang="la">
+            <div type="edition" xml:space="preserve" subtype="EDR" n="EDR138359" xml:lang="la" resp="#GC">
```

Comment:
delete the subtype and n attriubutes with the EDR references, as we're replacing the text